errortrace-doc/errortrace/scribblings/errortrace.scrbl

```diff
@@ -551,6 +565,28 @@ Same as @racket[stacktrace-imports^].}
 
 }
 
+
+@defsignature[stacktrace-filter^ ()]{
+
+The function defined for @racket[stacktrace-filter^] provides additional
+control over the expressions that are instrumented for debugging an
+profiling.
+
+@history[#:added "1.2"]
+
+@defproc[(should-annotate? [stx syntax?] [phase exact-nonnegative-integer?]) any/c]{
+
+Determines whether the @racketout[annotate] function from
+@racket[stacktrace/filter@] or @racket[stacktrace/annotator/filter@]
+adds debugging or profiling annotation to the expression represented
+by @racket[stx]. Annotation is added only when
+@sigelem[stacktrace-filter^ should-annotate?] returns a true value.
+
+The default filter function used by @racket[stacktrace@] and
+@racket[stacktrace/annotator@] annotates an expression if it has a
+source location.}}
```

Comment:
I would like `should-annotate?` to apply to test coverage: cover can take advantage of it :).

I don't really know anything about the profiling implementation, so no opinions there.